### PR TITLE
Move screen related code to the folder 'screens'.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

**Review by kenzieschmoll — COMMENTED** [review]
sort the imports - then lgtm

File: packages/devtools_app/lib/src/app.dart [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 // @dart=2.9
 
+// ignore_for_file: directives_ordering
```
[the remark this answers is not in the document]
Comment (polina-c) [reply packages/devtools_app/lib/src/app.dart]: oh, I think I clicked wrong button and did not notice. Thanks for the catch. Fixed.

File: packages/devtools_app/lib/src/config_specific/import_export/import_export.dart [hunk 2]
```diff
@@ -4,13 +4,15 @@
 
 // @dart=2.9
 
+// ignore_for_file: directives_ordering
```
[the remark this answers is not in the document]
Comment (polina-c) [reply packages/devtools_app/lib/src/config_specific/import_export/import_export.dart]: done

File: packages/devtools_app/lib/src/service/service_manager.dart [hunk 3]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: import_of_legacy_library_into_null_safe
+// ignore_for_file: import_of_legacy_library_into_null_safe, directives_ordering
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/service/service_manager.dart:5]: remove the ignore for directives_ordering, here and elsewhere. I just searched through this PR for "directives_ordering" to find all instances.

**Review by kenzieschmoll — APPROVED** [review]

**Pull request merged**